modules/microshift-install-rpms.adoc

```diff
@@ -20,28 +20,26 @@ Use the following procedure to install {microshift-short} from an RPM package.
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
-    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+    --enable rhocp-${ocp-version}-for-rhel-${op-system-version-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-rhel-${op-system-version-major}-$(uname -m)-rpms # <2>
 ----
-<1> Replace _<4.18>_ and _<9>_ with the compatible versions of your {microshift-short} and {op-system-base-full}.
-<2> Replace  _<9>_ with the compatible version of {op-system-base}.
 
 . For extended support (EUS) releases, also enable the EUS repositories by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-`$ sudo subscription-manager repos \
-    --enable rhel-<9>-for-x86_64-appstream-eus-rpms \ # <1>
-    --enable rhel-<9>-for-x86_64-baseos-eus-rpms` # <2>
+$ sudo subscription-manager repos \
+    --enable rhel-{op-system-version-major}-for-x86_64-appstream-eus-rpms \ # <1>
+    --enable rhel-{op-system-version-major}-for-x86_64-baseos-eus-rpms` # <2>
 ----
-<1> Replace _<9>_ with the compatible major version number of {op-system-base}.
-<2> Replace _<9>_ with the compatible major version number of {op-system-base}.
+<1> Use the compatible major version number of {op-system-base}.
+<2> Use the compatible major version number of {op-system-base}.
```

Comment:
I think (1) and (2) just need to be removed. The doc will render into the compatible version of OS.
It's a major version, so...

Reply:
Right, but we anticipate supporting two major versions, correct? So the footnotes can stay. Docs does not need to keep updating this every release.

Reply:
True, true.